packages/ng-schematics/src/ng-new/index.ts

```diff
@@ -148,7 +141,7 @@ export function newProject(options: OptionsSchema): Rule {
 					installChain.push(gitTask);
 				}
 
-				if (!options.skipInstall && !allOptionsProvided) {
+				if (!options.skipInstall) {
```

Comment:
This should still check for name provided. If you haven't been through the choose action loop, we shouldn't start the project as well